recipe/meta.yaml

```diff
@@ -23,10 +23,14 @@ requirements:
     - cython
     - numpy
     - libgdal
+    # rasterio's code for loading gdal DLL doesn't work in libgdal 3.1.3
+    - libgdal <3.1.3  # [win and py<=36]
   run:
     - python
     - setuptools >=0.9.8
     - libgdal
+    # rasterio's code for loading gdal DLL doesn't work in libgdal 3.1.3
+    - libgdal <3.1.3  # [win and py<=36]
```

Comment:
@conda-forge/rasterio, not sure how you feel about this.  I will try to reproduce this problem in a simpler from and report it upstream soon.

Reply:
Hmm, we'll see what happens but the following worked on my Windows virtual box without a problem:
```
conda create -y -n test python=3.6 libgdal=3.1.3 rasterio=1.1.7
conda activate test
python -c "import rasterio"
```

Reply:
Okay, this didn't work.  I'm removing this change.